src/main/resources/openapiv2.json

```diff
@@ -0,0 +1,889 @@
+                    }
+                }
+            },
+            "parameters": [
+                {
+                    "name": "groupid",
+                    "in": "path",
+                    "description": "ID of the consumer group to which the subscribed consumer belongs.",
+                    "required": true,
+                    "type": "string"
+                },
+                {
+                    "name": "name",
+                    "in": "path",
+                    "description": "Name of the subscribed consumer.",
+                    "required": true,
+                    "type": "string"
+                }
+            ]
+        },
+        "/consumers/{groupid}/instances/{name}/positions/end": {
+            "post": {
+                "tags": [
+                    "Consumers"
+                ],
+                "description": "Configures a subscribed consumer to seek (and subsequently read from) the last offset in one or more given topic partitions. This is the default behavior.",
+                "operationId": "seekToEnd",
+                "consumes": [
+                    "application/json"
+                ],
+                "parameters": [
+                    {
+                        "name": "body",
+                        "in": "body",
+                        "description": "List of topic partitions to which the consumer is subscribed. The consumer will seek the last offset in the specified partitions.",
+                        "required": true,
+                        "schema": {
+                            "$ref": "#/definitions/PartitionsForPositions"
+                        }
+                    }
+                ],
+                "responses": {
+                    "204": {
+                        "description": "No content."
+                    },
+                    "404": {
+                        "description": "The specified consumer instance was not found."
+                    }
+                }
+            },
+            "parameters": [
+                {
+                    "name": "groupid",
+                    "in": "path",
+                    "description": "ID of the consumer group to which the subscribed consumer belongs.",
+                    "required": true,
+                    "type": "string"
+                },
+                {
+                    "name": "name",
+                    "in": "path",
+                    "description": "Name of the subscribed consumer.",
+                    "required": true,
+                    "type": "string"
+                }
+            ]
+        },
+        "/consumers/{groupid}/instances/{name}/records": {
+            "get": {
+                "tags": [
+                    "Consumers"
+                ],
+                "description": "Retrieves information about the records processed by a subscribed consumer, including message values, topics, and partitions. Uses the `subscribe` or `assign` API.",
```

Comment:
It's not about retrieving information about records ... it's about retrieving the records/messages from the topic partitions on which the consumer is subscribed to. @laidan6000 @PaulRMellor